### ISSUE TITLE
VS2022 support

### ISSUE THREAD

**Issue by Trokkin** [emartineau/SelectionWrapper#12]
Pretty self-explanatory. The only version supported currently is vs2019.

It should only be a matter of changing manifesto, and there are already 3 forks doing that.

**Comment by lfr** [comment]
Yes please, I've offered a workaround [here](https://github.com/emartineau/SelectionWrapper/issues/11#issuecomment-982508966) but it would be nice to have the one from the marketplace fixed.

**Comment by emartineau** [comment]
Sorry for the absence! I don't use Visual Studio much these days, which was the primary motivator for my work on this extension. Not to mention the documentation for making VS extensions is... lacking. So I tend to and only check on this every few months or so.

I was hoping that VS 2022 would have added this feature... but evidently that didn't happen. Anyways feel free to fork or make pull requests! It's why this repo is public.

You'll find the latest release on this repo (which is pretty much  just a refactor) includes a .vsix for VS 2022. VS 2022 apparently brings in some extra work compared to past iterations when it comes to adding support to existing extensions. Nothing big in this case, but it does mean separate .vsix packages. And currently the VS marketplace only allows 1 VSIX package per listing. [Allegedly this'll change in the future](https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022#publish-your-extension), but until then, I'll probably just be putting any VS 2022-specific releases on this repo.

I doubt I'll be contributing much to this project, but any pull requests are welcome and could then get passed to the current 3k+ people with the extension on VS2019.

**Comment by lfr** [comment]
I don't code that much anymore these days, hopefully someone else will come along with a little more availability to keep this alive. Considering the previous version's code just works on the new one, I think it won't require that many (if any) updates for the foreseeable future.
